cJSON.h

```diff
@@ -81,84 +81,113 @@ typedef struct cJSON_Hooks
       void (*free_fn)(void *ptr);
 } cJSON_Hooks;
 
+#if !defined(__WINDOWS__) && (defined(WIN32) || defined(WIN64) || defined(_MSC_VER) || defined(_WIN32))
+#define __WINDOWS__
+#endif
+#ifdef __WINDOWS__
+#if defined(CJSON_DECLARE_STATIC)
+#define CJSON_DECLARE(type)			type __stdcall
+#define CJSON_DECLARE_NONSTD(type)	type __cdecl
+#define CJSON_DECLARE_DATA
```

Comment:
Remove `CJSON_DECLARE_NONSTD` and `CJSON_DECLARE_DATA`. (also from other branches of the if)